lametro/wagtail_hooks.py

```diff
@@ -26,11 +28,34 @@
 )
 
 
+class AlertFilterSet(django_filters.FilterSet):
+    EXPIRED_CHOICES = (
+        ("expired", "Expired"),
+        ("all", "All Alerts"),
+    )
+
+    expired = django_filters.ChoiceFilter(
+        label="Alert Status",
+        method="filter_expired_alerts",
+        choices=EXPIRED_CHOICES,
+        empty_label="Not Expired",
+    )
+
+    def filter_expired_alerts(self, queryset, name, value):
+        if value == "expired":
+            return Alert.objects.filter(expiration__lte=timezone.now())
+        elif value == "all":
+            return Alert.objects.all()
+        else:
+            return queryset
```

Comment:
I kept the `else` for readability's sake, but removing it would still return the default already evaluated queryset.